json-rpc/src/mock_db.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
nit: make separate folder for tests and move mock_db and existing coverage there for better isolation